scripts/gdal/gdal_helper.py

```diff
@@ -19,11 +19,11 @@ class GDALExecutionException(Exception):
     pass
 
 
-class EpsgCode(str, Enum):
-    EPSG_2193 = "EPSG:2193"
-    """ NZGD2000 / New Zealand Transverse Mercator 2000 (NZTM) """
-    EPSG_4326 = "EPSG:4326"
-    """ WGS84 - World Geodetic System 1984"""
+class EpsgNumber(int, Enum):
+    EPSG_NZGD2000 = 2193
```

Comment:
NZGD2000 is [EPSG:4167](https://epsg.io/4167)

Reply:
I was just basing this on the original comment. Would `NZTM` be a better name?

Reply:
See #1063.